streaming/pom.xml

```diff
@@ -107,33 +107,6 @@
           <failIfNoTests>false</failIfNoTests>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
It is unnecessary plugin added here. There is no scala tests added in this module. Because of it there are duplicate test count is added in CI thats why it is removed.